### DISCUSSION TITLE
Invoice Download in Fluxstore applications

### DISCUSSION THREAD

**Discussion by shubhamrane48** [inspireui/support#5241] [Working on Features]
I found 2 plugins which support woocommerce for PDF invoice on each order can you please suggest which is supported for fluxstore.

Plugins-
1. https://woocommerce.com/products/pdf-invoices/
2. https://wordpress.org/plugins/woocommerce-pdf-invoices-packing-slips/

**Comment by minhcasi** [comment]
move to #5253
